### ISSUE TITLE
Circularize (FASTA only) checkbox should only appear when file format is FASTA or "Guess from extension"

### ISSUE THREAD

**Issue by dgruano** [manulera/OpenCloning_frontend#251]

**Comment by manulera** [comment]
Good point!

**Comment by dgruano** [comment]
The problem with the current implementation is that if a crazy user (like me) first clicks the circularize checkbox and then changes source type, the circularize value is still set and this returns an error:

<img width="371" alt="Screenshot 2024-11-26 at 22 20 35" src="https://github.com/user-attachments/assets/1fedf799-ed6a-4385-ba8e-b9c2d39c0c06">

We should uncheck the box within the onChange function of the dropdown.

**Comment by manulera** [comment]
Good call! Anyway, from Valerie's email this should probably be allowed for all types of files since some programs may not import the topology information correctly.

**Comment by dgruano** [comment]
There is only one type for which I think it does not make sense: history files.
